docs/installing-metabase.md

```diff
@@ -0,0 +1,40 @@
+# Installing Metabase
+
+Welcome to Metabase! We’re excited to help you get started. We've made our install process as simple as we can manage to get you right to your data. 
+
+Metabase is built and packaged as a Java jar file and can be run anywhere that Java is available (after installation, Metabase is accessed via a browser).
```

Comment:
Looks like you use `JAR` instead of `jar` in the following paragraph. I believe `JAR` is technically correct since it's an acronym.

Reply:
Hah, I stole that directly from another part of the docs, I'll update it there too in a different PR.

Reply:
Oh yeah, I'm 100% sure we're not at all consistent about that, haha. I'm not religious about `JAR` vs. `jar` vs `.jar`, but I do think we should strive for consistency. `JAR` looks stuffier, so if you want to go with `jar` I'm cool with that.